assets/app/scripts/services/logLinks.js

```diff
@@ -50,19 +50,42 @@ angular.module('openshiftConsole')
         newTab(params);
       };
 
-      // Not currently used.
-      // new tab: path/to/current?view=textonly
-      var textOnlyLink = function() {
-        newTab({view: 'textonly'});
+      // Kibana archive URL v1: using a string & _.template
+      var archiveTpl = [
+        '<%= baseurl %>',
+        '/#/discover?',
+        '_g=',
+        '(refreshInterval:(display:Off,pause:!f,section:0,value:0),',
+        'time:(from:now-7y,mode:quick,to:now))&_a=(columns:!(_source),',
+        'filters:!((meta:(disabled:!f,index:\'<%= namespace %>.*\',key:kubernetes_namespace_name,negate:!f,value:<%= namespace %>),',
+        'query:(match:(kubernetes_namespace_name:(query:<%= namespace %>,type:phrase)))),',
+        '(meta:(disabled:!f,index:\'<%= namespace %>.*\',key:kubernetes_pod_name,negate:!f,value:<%= podname %>),',
+        'query:(match:(kubernetes_pod_name:(query:<%= podname %>,type:phrase))))),',
+        'index:\'<%= namespace %>.*\',interval:auto,query:\'\',sort:!(time,desc))'
+      ].join('');
+
+      // takes { url: '', namespace: '', podname: '' }
+      var archiveUri = _.template(archiveTpl);
+
+      // Kibana archive URL v2: using URI.js.
+      // Still working on this impl, URI doesn't like the hash before the query params & requires a plugin to manipulate in this manner...
+      var archiveUri_v2 = function(opts) {
+        var uri = new URI(opts.baseurl+'/#/discover?');
+        uri.addFragment('_g','(refreshInterval:(display:Off,pause:!f,section:0,value:0),time:(from:now-7y,mode:quick,to:now))');
+        uri.addFragment('_a', "(columns:!(_source),filters:!((meta:(disabled:!f,index:'"+opts.namespace+".*',key:kubernetes_namespace_name,negate:!f,value:'+opts.namespace+'),query:(match:(kubernetes_namespace_name:(query:'+opts.namespace+',type:phrase)))),'(meta:(disabled:!f,index:'"+opts.namespace+".*',key:kubernetes_pod_name,negate:!f,value:"+opts.podname+"),query:(match:(kubernetes_pod_name:(query:'+opts.podname+',type:phrase))))),index:'"+opts.namespace+".*',interval:auto,query:'',sort:!(time,desc))");
+        return uri.toString();
```

Comment:
alternately you can compute the fragment url in one step, then set it as the fragment of the real url in a second step